releases/v2.1.0-alpha.20180730.md

```diff
@@ -0,0 +1,263 @@
+- `SET` now properly rejects attempts to use
+  invalid variable names starting with `tracing.`. [#27216][#27216]
+- Fixed `NULL` equality handling in the experimental lookup join feature. [#27336][#27336]
+- `ALTER ... EXPERIMENTAL CONFIGURE ZONE` is now properly tracked in statement statistics. [#27213][#27213]
+- Invalid uses of set-generating functions in `FROM` clauses are now reported with the same error code as PostgreSQL. [#27390][#27390]
+- The number of `COPY` columns is now correctly verified during `IMPORT ... PGDUMP`. [#27345][#27345]
+- `CHANGFEED`s now correctly emit all versions of quickly changing rows. [#27612][#27612]
+- Alleviated a scenario in which a large number of uncommitted Raft commands could cause memory pressure at startup time. [#27009][#27009]
+- Prevented the unbounded growth of the Raft log caused by a loss of quorum. [#27774][#27774]
+- Foreign key references in `IMPORT ... PGDUMP` are now processed in the correct order. [#27782][#27782]
+- Fix the "global matching" regex flag to `g`. [#27128][#27128] {% comment %}doc{% endcomment %}
+
+### Performance Improvements
+
+- Transactional writes are now pipelined when being replicated and when being written to disk, dramatically reducing the latency of transactions that perform multiple writes. This can be disabled using the new `kv.transaction.write_pipelining_enabled` [cluster setting](../v2.1/cluster-settings.html). [#26599][#26599]
+- Reduced CPU utilization in clusters with many ranges, also during periods of lease rebalancing. [#26910][#26910] [#26907][#26907]
+- Reduced the memory size of commonly used Request and Response objects. [#27112][#27112]
+- Improved low-level iteration performance. [#27299][#27299]
+- Prevented a scenario when dropping a table that could cause excessive compaction activity that would degrade performance significantly. [#27353][#27353]
+- Limited the scanner for running incessantly on stores with 100s of thousands of replicas. [#27441][#27441]
+- Prevented dead nodes in clusters with many ranges from causing unnecessarily high CPU usage. [#26911][#26911]
+- Significantly reduce CPU usage when a large number of ranges are deleted from a node. [#27520][#27520]
+- Min, max, sum, and avg now take linear time when used for aggregation as window functions for all supported window frame options. [#26988][#26988]
+- `CHANGEFEED`s no longer hold all data for each poll in memory at once, increasing scalability. [#27612][#27612]
+
+### Build Changes
+
+- Upgraded protobuf to 3.6.0 [#26935][#26935]
+
+### Doc Updates
+
+Docs team: Please add these manually.
+
+### Contributors
+
+This release includes 328 merged PRs by 42 authors. We would like to thank the following contributors from the CockroachDB community, with special thanks to first-time contributors Andrew Woods, Art Nikpal, Ivan Kozik, Madhav Suresh, Tarek Badr, and nexdrew.
+
+- Andrew Woods
+- Art Nikpal
+- Brett Snyder
+- Emmanuel
```

Comment:
make this `Emmanuel Sales`.

Reply:
Also perhaps remove Andy Woords and Emmanuel from this list since we're not including CRL team members here for the time being.

Reply:
Done.